src/commands/install-node.yml

```diff
@@ -59,11 +59,36 @@ steps:
         - run:
             name: Install and use node version
             command: |
-              # if("<<parameters.version>>" -eq "latest") {
-              #   choco install nodejs
-              # } else {
-              #   $r = Invoke-WebRequest https://nodejs.org/download/release/index.json | ConvertFrom-Json
-              #   $latest_of_version = ($r | where { $_.version -like "v<<parameters.version>>*" })[0].version.substring(1)
-              #   choco install nodejs --version $latest_of_version --allow-downgrade
-              # }
+              $nodeIndex = "https://nodejs.org/download/release/index.json"
+              $registry = (Invoke-WebRequest $nodeIndex).content
+
+              if ("<<parameters.version>>" -eq "latest") {
+                $latest = $registry | jq -c -r '.[0].version | ltrimstr(\"v\")'
+
+                nvm install $latest
+                nvm use $latest
+              } elseif ("<<parameters.version>>" -eq "lts") {
+                $lts = $registry |
+                    jq -c -r 'first(.[] | select(.lts != false)).version | ltrimstr(\"v\")'
+
+                nvm install $lts
+                nvm use $lts
```

Comment:
`nvm-windows` [added support for "latest, "lts" on v1.1.8 ](https://github.com/coreybutler/nvm-windows/releases/tag/1.1.8) but circleci still uses v1.1.7 so we still need to use jq to get the right version.